src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -442,12 +443,17 @@
             activeFieldsBySObject[sObjectName].forEach(function(currentField) {
                 currentFieldGroup.fields.push(currentField);
                 currentFieldGroup.sObjectLabel = currentField.sObjectLabel;
+                donationMatchingOptions.push({
+                    label: currentField.label,
+                    value: currentField.name
+                });
             });
 
             batchFieldOptions.fieldGroups.push(currentFieldGroup);
 
         });
         component.set('v.batchFieldOptions', batchFieldOptions);
+        component.set('v.batchInfo.donationMatchingOptions', donationMatchingOptions);
```

Comment:
How were we setting donationMatchingOptions previously? Was it just getting a copy of the batch fields somewhere?

Reply:
(UPDATE: I think I found it, it's getting all the fields from getRelevantFields() in the apex controller and then previous to this work, just sticking with that list)

Reply:
That's what I've just found as well

Reply:
If we're setting it here only, and we always call this function in the course of the wizard, we don't need to pass back anything from apex, so we can actually remove donationMatchingOptions from the model and remove the whole method that sets it in Apex.